rest_framework/validators.py

```diff
@@ -33,7 +33,16 @@ def qs_filter(queryset, **kwargs):
         return queryset.none()
 
 
-class UniqueValidator(object):
+class ContextBasedValidator(object):
+    """Base class for validators that need a context during evaluation.
+
+    In extension to regular validators their `__call__` method must not only
+    accept a value, but also an instance of a serializer.
```

Comment:
I think it should say "serializer field" here?

Reply:
I'm not sure.  Depends on the validator, I guess.  I think it's mostly relevant for validators used on serializers since fields are deepcopied.